docs/faststore/docs/reference/project-structure/updating-the-cli-package-version.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: "Updating the '@fastore/cli' package version"
+---
+
```

Comment:
suggestion: I think we can also add the info that since `@faststore/cli": "^3.0.87` @faststore/cli version depends on the @faststore/core package and not the opposite, we can mention this [release note](https://developers.vtex.com/updates/release-notes/2024-09-11-faststore-cli-dependency). WDYT?

Reply:
actually is the contrary 👍 added the suggestion! Thanks!